test/blackbox/PubSubReader.hpp

```diff
@@ -33,7 +33,6 @@
 #include <fastrtps/xmlparser/XMLTree.h>
 #include <fastrtps/utils/IPLocator.h>
 #include <fastrtps/transport/UDPv4TransportDescriptor.h>
-
```

Comment:
This blank line separates logical #include units. Why did you remove it?

Reply:
Changed